.storybook/load-stories.js

```diff
@@ -1,22 +1,21 @@
 /*
 Copyright (c) 2018-2019 Uber Technologies, Inc.
-
 This source code is licensed under the MIT license found in the
 LICENSE file in the root directory of this source tree.
 */
 /* global module */
 /* eslint-disable flowtype/require-valid-file-annotation */
 
+import React from 'react';
 import {storiesOf} from '@storybook/react';
 
 import scenarios from '../src/**/*.scenario.js';
 
-scenarios.reduce(
-  (stories, scenario) => stories.add(scenario.name, scenario.component),
-  storiesOf('baseui', module),
-);
+const light = storiesOf('baseui', module);
+const dark = storiesOf('baseui-dark', module);
 
-scenarios.reduce(
-  (stories, scenario) => stories.add(scenario.name, scenario.component),
-  storiesOf('baseui-dark', module),
-);
+scenarios.forEach(scenario => {
+  const Component = scenario.component;
+  light.add(scenario.name, () => <Component />);
+  dark.add(scenario.name, () => <Component />);
+});
```

Comment:
do we want to include this changeset in this PR?

Reply:
I don't see an issue with doing so.. I wanted to get hooks working with storybook so that the scenarios in this pr can run load

Reply:
sounds good, I wasn't sure why we added it here - thanks for the clarification Chase!

Reply:
shows up in this pr because I added it to the pr :)